File: Algoritmo(C)/Alg_S8_3.c
#include <stdio.h>
int main(){
    //variables
    int vetor[10], i;
    //input
    for(i = 0; i <= 9; i++){
        printf("Digite o %do numero: ",i);
        scanf("%d",&vetor[i]);
    }
    for(i = 9; i >= 0; i--){
        printf("%d\n",vetor[i]);
    }
}

File: Algoritmo(Python)/Alg_S8_3.py
#variables
vetor = []
#input
for(i) in range(0, 10):
    num = int(input("Digite um número para o vetor {0}/10: ".format(i + 1)))
    vetor.append(num)
vetor.reverse() #inverte a ordem
for(i) in (vetor):
    print(i)